tests/publiccloud/run_ltp.pm

```diff
@@ -86,6 +86,7 @@ sub run {
     my $ltp_repo = get_var('LTP_REPO', 'https://download.opensuse.org/repositories/benchmark:/ltp:/stable/' . generate_version("_") . '/');
     my $provider;
     my $instance;
+    my $ltp_package = 'ltp-stable';
```

Comment:
just a small thing. I dont see any good reason to define the package here. Would it make sense to have it public for this module? in such case the `gen_ltp_env` wouldnt need to change. wdyt?